Jenkinsfile

```diff
@@ -1,14 +1,25 @@
-@Library('global-jenkins-library@1.9.0') _
+@Library('global-jenkins-library@2.0.0') _
```

Comment:
You may want to upgrade to 2.1.0, even if there was no modification in `buildSimpleDocker_v3`.

Reply:
Thanks, see https://github.com/iExecBlockchainComputing/python-hello-world/pull/13/commits/97f476a775e673cc3b18910f176903e2af4c3ffa :)